CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
+## 0.4.7
+ This is a feature release.
+
+* [TK-312] Unbuffered streams in Java to match clojure support released in 0.4.5.
```

Comment:
You don't need to add a ticket for the other one, just saying we should add the link for 312.